data/bkpecho.json

```
@@ -1,64 +1,196 @@
+            "url": "https://twitter.com/bkpecho",
+            "icon": "FaTwitter"
+        },
+        {
+            "group": "Socials",
+            "name": "LinkedIn: Let's Connect!",
+            "url": "https://www.linkedin.com/in/bkpecho/",
+            "icon": "FaLinkedin"
+        },
+        {
+            "group": "Coding Challenges",
+            "name": "LeetCode: The World's Leading Online Programming Learning Platform",
+            "url": "https://leetcode.com/bkpecho/",
+            "icon": "SiLeetcode"
+        },
+        {
+            "group": "Coding Challenges",
+            "name": "Codewars: Achieve Code Mastery Through Challenge",
+            "url": "https://codewars.com/users/bkpecho",
+            "icon": "SiCodewars"
+        },
+        {
+            "group": "Coding Challenges",
+            "name": "FrontEnd Mentor: Front-end Coding Challenges",
+            "url": "https://frontendmentor.io/profile/bkpecho",
+            "icon": "SiFrontendmentor"
+        },
+        {
+            "group": "My Top Handpicked GitHub Teasures",
+            "name": "Best GitHub Repos To Master JavaScript",
+            "url": "https://github.com/stars/bkpecho/lists/8-github-repos-to-master-js",
+            "icon": "SiJavascript"
+        },
+        {
+            "group": "My Top Handpicked GitHub Teasures",
+            "name": "Must-Have React Repos on Github",
+            "url": "https://github.com/stars/bkpecho/lists/must-have-react-repos-on-github",
+            "icon": "FaReact"
+        },
+        {
+            "group": "My Top Handpicked GitHub Teasures",
+            "name": "Useful GitHub Repositories for Web Developers",
+            "url": "https://github.com/stars/bkpecho/lists/useful-github-repos-for-web-devs",
+            "icon": "FaDatabase"
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Graduated from Cavite State University",
+            "date": "April 2023",
+            "icon": "FaGraduationCap",
+            "color": "#3498db",
+            "description": "Completed my Bachelor of Science in Information Technology degree from Cavite State University.",
+            "url": "https://twitter.com/bkpecho/status/1651898479759994881?s=20"
+        },
+        {
+            "title": "Joined EddieHub Community",
+            "date": "March 2023",
+            "icon": "FaUsers",
+            "color": "#e74c3c",
+            "description": "Became a member of the EddieHubCommunity, a welcoming and inclusive community that encourages collaboration and contributions to open source projects.",
+            "url": "https://www.eddiehub.org/"
+        },
+        {
+            "title": "Made First Open Source Contribution",
+            "date": "March 2023",
+            "icon": "FaCodeBranch",
+            "color": "#2ecc71",
+            "description": "Made my first open source contribution to a project on GitHub.",
+            "url": ""
```

Here the URL key is optional. You can remove these from lines no `105`, `145`, `161`, `169`, `177`, `185` & `194`. 

Instead of taking visitors to a 404, it's better to make it unclickable. 

```suggestion
            "description": "Made my first open source contribution to a project on GitHub."
```

---

Thanks for the review! @kumarsonsoff3 Will do.